### PR TITLE
Bump to v4.2.1

### PR DESCRIPTION
**Pull request by mdo**
I messed up in the publishing of our npm package, so have to bump to a new release to publish once again. Say hello to v4.2.1!

### CODE REVIEW

**Comment by mdo** [comment]
Okay, after see different diffs between myself and @XhmikosR, I think we're good to go. Waiting on his final approval before squashing and merging, and then doing the full publish dance.

**Review by XhmikosR — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
Hh

**Pull request merged**
